src/SwedbankPay.Sdk.Infrastructure/JsonSerialization/JsonSerialization.cs

```diff
@@ -0,0 +1,57 @@
+﻿using SwedbankPay.Sdk.JsonSerialization.Converters;
+using SwedbankPay.Sdk.PaymentOrders;
+using System.Text.Json;
+
+namespace SwedbankPay.Sdk.JsonSerialization
+{
+    public static class JsonSerialization
+    {
+        private static JsonSerializerOptions settings = null;

```

Comment:
Codacy found an issue: [Remove this initialization to 'settings', the compiler will do that for you.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)